Assets/Scripts/SpawnManager.cs

```diff
@@ -5,18 +5,41 @@
 public class SpawnManager : MonoBehaviour
 {
     public GameObject enemyPrefab;
-    private float spawnRange = 9.0f;
+    public GameObject powerupPrefab;
```

Comment:
because this gameobject is public you should name it in PascalCase -> PowerupPrefab